book/03_Geospatial_data_files/01_Geospatial_data_formats.md

```diff
@@ -0,0 +1,53 @@
+# Geospatial data formats
+
+## Vector data formats
+
+In this tutorial we will be using the following vector data formats:
+
+### 1. SHP
+
+Shapefile is the most widely known format for distributing geospatial data. It is a standard first developed by ESRI almost 30 years ago, which is considered ancient in software development.^3^ 
+
+The Shapefile consists of several files: in addition to one file with the actual geometry data, another file for defining the coordinate reference system is needed, as well as a file for defining the attributes and a file to index the geometries. This makes operating Shapefiles slightly clunky and confusing. However, Shapefile has been around for so long that any GIS software supports handling it.^3^
+
+Internally, Shapefile uses Well-known binary (WKB) for encoding the geometries. This is a compact format that is based on tabular thinking, i.e. the row and column number of a value is significant. A minor nuisance is the limitation of the attribute field names to 10 characters and poor Unicode support, so some abbreviations and forcing to ASCII may have to be used.^3^
+
+For more details about the shapefiles you can refer to the [ESRI Shapefile Technical Description](https://www.esri.com/content/dam/esrisites/sitecore-archive/Files/Pdfs/library/whitepapers/pdfs/shapefile.pdf)
+
+### 2. GeoJSON
+
+GeoJSON is a subset of JSON (JavaScript object notation). It was developed 10 years ago by a group of enthusiastic GIS developers. The core idea is to provide a specification for encoding geospatial data while remaining decodable by any JSON decoder.^3^
+
+Being a subset of the immensely popular JSON, the parsing support is on a different level than with Shapefile. In addition to support from most GIS software, any web developer will be able to write a custom GeoJSON parser, opening new possibilities for integrating the data.^3^
+
+Being designed as one blob of data instead of a small "text database" means it is simpler to handle but is essentially designed to be loaded to memory in full at once.^3^ 
+
+For more information about GeoJSON data formats, you can refer to [https://geojson.org/](https://geojson.org/).
+
+## Raster data formats
+
+
+In this tutorial we will be using the following raster data format:
+
+### 1. GeoTiffs
+
+GeoTIFF is a public domain metadata standard that enables georeferencing information to be embedded within an image file. The GeoTIFF format embeds geospatial metadata into image files such as aerial photography, satellite imagery, and digitized maps so that they can be used in GIS applications.^2^
```

Comment:
+ Include an explicit mention that GeoTIFF is derived from [TIFF (Tagged Image File Format)](https://en.wikipedia.org/wiki/TIFF) and link to the relevant Wikipedia page.